tests/kernels/attention/test_flashmla.py

```diff
@@ -35,20 +35,20 @@ def cal_diff(x: torch.Tensor, y: torch.Tensor, name: str) -> None:
 @pytest.mark.parametrize("block_size", [64])
 @pytest.mark.parametrize("causal", [True])
 @pytest.mark.parametrize("varlen", [False, True])
+@pytest.mark.parametrize("dtype", [torch.bfloat16])
+@pytest.mark.parametrize("device", ["cuda:0"])
 @torch.inference_mode()
 def test_flash_mla(b, s_q, mean_sk, h_q, h_kv, d, dv, block_size, causal,
-                   varlen):
-    # TODO: parametrize using pytest
-    dtype = torch.bfloat16
-    device = torch.device("cuda:0")
+                   varlen, dtype, device):
```

Comment:
I mean leave the device `device = torch.device("cuda:0")` same, just parametrizing for dtype. And possibly adding more dtype